Lib/bdb.py

```diff
@@ -26,6 +34,12 @@ def __init__(self, skip=None):
         self.frame_returning = None
 
     def canonic(self, filename):
+        """Get a filename into a canonical form.
```

Comment:
Thanks for that explanation.  Looking through the patch history, `canonic` was added later.  I'll try to find it again and maybe the commit comment will help.  I have no idea how the old commits were ported to git, but it is just one more cool thing -- to have all the history available.  :-)